Makefile

```diff
@@ -81,16 +82,13 @@ lint: golangci-lint ## Run golangci-lint against code.
 	GOOS=$(TARGET_OS) CGO=1 $(GOLANGCI_LINT) run ./...
 
 .PHONY: check
-check: manifests generate add-license lint test
+check: manifests generate fmt addlicense lint test ## Generate manifests, code, lint, add licenses, test
```

Comment:
addlicense tool doesn't make sense for me. probably it is better replace it with check-license.

Reply:
ok. I will change it to check-license